src/qtt/instrument_drivers/gates.py

```diff
@@ -175,24 +190,31 @@ def _remove_gates(self):
             self.functions.pop(f'get_{gate}')
             self.functions.pop(f'set_{gate}')
 
-    def add_instruments(self, instruments):
+    def add_instruments(self, instruments: List[Instrument]):
```

Comment:
Policy is to either use type annotations in the entire file, or not at all.

Reply:
Removed for now, but where can I find this policy?